packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -222,7 +222,7 @@ class AttachCommand extends FlutterCommand {
         try {
           isolateDiscoveryProtocol = device.getIsolateDiscoveryProtocol(module);
           observatoryUri = Stream<Uri>.value(await isolateDiscoveryProtocol.uri).asBroadcastStream();
-        } catch (_) {
+        } on Exception catch (_) {
```

Comment:
nit. we don't need to catch unused Exception

Reply:
Done.